examples/sequence.rb

```diff
@@ -0,0 +1,37 @@
+require 'dotenv'
+Dotenv.load '.env-fixer_client'
+
+require 'fixer_client'
+Fixer.reset!
+
+Fixer.configure do |c|
+  c.client_id = ENV['FIXER_CLIENT_ID']
+  c.client_secret = ENV['FIXER_CLIENT_SECRET']
+  c.endpoint = 'http://fixer.prx.dev/api/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.